### PR TITLE
allow passing through a custom task definition file through to ecs_se…

### PR DESCRIPTION
**Pull request by jamesgorrie**
We need this as we want to use all the other service niceness, but also need to add an extra task (mongo).

I felt we could have done some fanciness around concatenating the array with extra services, but it felt clearer to just allow someone to redefine it, and copy what they want from the existing templates. 

This is a non breaking change.

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Not sure what to do about releasing to fix the build.

File: ecs/service/variables.tf [hunk 1]
```diff
@@ -133,3 +133,13 @@ variable "log_group_name_prefix" {
...
   default     = "platform"
 }
+
+variable "use_task_definition_template_path" {
+  description = "A task definition tmeplate to use"
+  default     = 0
+}
+
+variable "task_definition_template_path" {
+  description = "A task definition tmeplate to use"
```
Comment (alexwlchan) [inline ecs/service/variables.tf]: sp: "tmeplate"

File: ecs/service/variables.tf [hunk 2]
```diff
@@ -133,3 +133,13 @@ variable "log_group_name_prefix" {
   description = "Cloudwatch log group name prexix"
   default     = "platform"
 }
+
+variable "use_task_definition_template_path" {
+  description = "A task definition tmeplate to use"
```
Comment (alexwlchan) [inline ecs/service/variables.tf]: nit: this description is wrong.

File: ecs/service/ecs_task/data.tf [hunk 3]
```diff
@@ -1,4 +1,6 @@
 data "template_file" "definition" {
+  count = "${1 - var.use_task_definition_template_path}"
```
Comment (alexwlchan) [inline ecs/service/ecs_task/data.tf]: Perhaps we could get rid of the boolean, and use a ternary here? Something like:

```hcl
count = "${var.task_definition_template_path == "" ? 1 : 0}"
```

I’m slightly guessing at the syntax, but that might be neater.

**Comment by alexwlchan** [comment]
To fix the build, you need to write a release note (example [in another PR](https://github.com/wellcometrust/terraform-modules/blob/c0b4728ce65bf09d4587ecebfd2c9f3170b6307c/RELEASE.md)) that starts with the release type, then describes the change:

```
RELEASE_TYPE: (patch | minor | major)

The description goes here.
```

Since this is a backwards-compatible API change, probably patch is most appropriate here.

On merging to master, this is used to bump the version number and assemble the new changelog.

File: ecs/service/variables.tf (hunk 2 above)
Comment (jamesgorrie) [reply ecs/service/variables.tf]: Copy paste error.

**Pull request merged**